components/common/src/main/java/com/hotels/styx/common/Strings.java

```diff
@@ -15,17 +15,17 @@
  */
 package com.hotels.styx.common;
 
-import com.google.common.base.Joiner;
+public final class Strings {
 
-/**
- * Common uses of Guava's Joiner class.
- */
-public final class Joiners {
-    public static final Joiner JOINER_ON_COMMA = Joiner.on(",").skipNulls();
-    public static final Joiner JOINER_ON_SEMI_COLON = Joiner.on(";").skipNulls();
-    public static final Joiner JOINER_ON_SPACE = Joiner.on(" ").skipNulls();
-    public static final Joiner JOINER_ON_NEW_LINE = Joiner.on("\n").skipNulls();
+    public static final boolean isNullOrEmpty(String s) {
```

Comment:
If this is sourced from Guava, perhaps it should be attributed accordingly?

Reply:
In all honesty, I have been writing implementations of this method for many years. This one differs from guava, in that it uses `s.isEmpty()` rather than `s.length() == 0`.